embedded-kafka/src/main/java/com/playtika/test/kafka/configuration/KafkaContainerConfiguration.java

```diff
@@ -212,4 +228,48 @@ public KafkaTopicsConfigurer kafkaConfigurer(
         return new KafkaTopicsConfigurer(kafka, zookeeperProperties, kafkaProperties);
     }
 
-}
\ No newline at end of file
+    /**
+     * Create folder (and parent folder if necessary) with write permissions for current user
+     *
+     * @param path folder to create
+     */
+    private void createPathAndParentOrMakeWritable(Path path) {
+        Stream.of(path.getParent(), path).forEach(p -> {
+            if (p.toFile().isDirectory()) {
+                makeWritable(p);
+            } else {
+                try {
+                    log.info("Create writable folder: {}", p);
+                    Files.createDirectory(p, PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwxrwxrwx")));
+                } catch (FileAlreadyExistsException e) {
+                    makeWritable(p);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
if we cannot create directory for zk or kafka for whatever reason -- container is not usable. instead of printing stacktrace please rethrow an exception wrapping it into runtimeexception.

Reply:
ok